packages/livechat/src/routes/TriggerMessage/container.js

```diff
@@ -8,15 +8,16 @@ import TriggerMessage from './component';
 
 
 export class TriggerMessageContainer extends Component {
-	handleStart() {
+	handleStart(props) {
 		parentCall('setFullScreenDocumentMobile');
 		parentCall('openWidget');
+		props.onRestore();
 		route('/');
 	}
 
 	render = (props) => {
 		parentCall('resetDocumentStyle');
-		return <TriggerMessage onStartChat={this.handleStart} {...props} />;
+		return <TriggerMessage onStartChat={() => this.handleStart(props)} {...props} />;
```

Comment:
<img width="638" alt="Screen Shot 2022-05-05 at 9 35 32 AM" src="https://user-images.githubusercontent.com/20868078/166925651-9750115d-0392-4f54-91c6-3c38d2600e62.png">
 @cauefcr @KevLehman

Reply:
think we should be good 👀